addons/payment/security/ir.model.access.csv

```diff
@@ -1,5 +1,4 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-payment_acquirer_all,payment.acquirer.all,model_payment_acquirer,,1,0,0,0
 payment_acquirer_system,payment.acquirer.system,model_payment_acquirer,base.group_system,1,1,1,1
 payment_transaction_all,payment.transaction.all,model_payment_transaction,,1,0,0,0
```

Comment:
I think that for all the payment models, we should stick to the new guidelines that we apply on our applications (at least in /dev/null).

- Classic CRUD right for internal users according to the use case (common used records with no critical data)
- Advanced CRUD for "managers" (accountant, sales, ...) for low-level records for control/maintenance purpose
- Total CRUD for system users, for debugging purpose
- No CRUD for portal/public users, use sudo according to the case to explicitly expose the data we want.

Reply:
Sounds good to me. The security rules of the payment module is still a big TODO. We'll need to chat a bit about that.